### PR TITLE
TECHDBT 1126 - Display update links on browsers <= IE 8

### PR DESCRIPTION
**Pull request by jessedoyle**
:elephant:
* Added `browser.modern?` call to ama_layout.
* A `before_action` call is now added to the host's ApplicationController when included.
* Added specs to test behaviour with different user agents.
* Added `.DS_Store` to `.gitignore`.
* Added my name and email to gemspec.
* Updated the minor version as this change includes new functionality.

We'll have to send PRs to the remove this functionality from the various apps if this is accepted.

https://ama-digital.myjetbrains.com/youtrack/issue/TECHDBT-1126

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:book:

File: lib/ama_layout/controllers/action_controller.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  module ActionController
+    extend ActiveSupport::Concern
+
+    included do
+      before_action :check_browser
+    end
+
+    private
+
+      def check_browser
```
Comment (MathieuGilbert) [inline lib/ama_layout/controllers/action_controller.rb]: Could you indent this method 1 less? Also no extra spaces for the assignment.

**Comment by MathieuGilbert** [comment]
:hankey: Looks great, just those minor formatting issues. It's easier for everyone if we follow the same style conventions.

File: lib/ama_layout/controllers/action_controller.rb (hunk 1 above)
Comment (jessedoyle) [inline lib/ama_layout/controllers/action_controller.rb]: Absolutely! (Old habits I guess)

**Comment by jessedoyle** [comment]
:wave:

**Comment by rubene** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**